xbmc/FileItem.cpp

```diff
@@ -1730,6 +1735,7 @@ void CFileItem::UpdateInfo(const CFileItem &item, bool replaceLabels /*=true*/)
       m_videoInfoTag = new CVideoInfoTag;
     }
 
+    SetDynPath(item.GetDynPath());
```

Comment:
This will be done already later in this method, this can be removed here.

Reply:
Not sure why there is this change (it's meaningless to me). I tend to believe it was introduced when merged from master before. Will remove it with new iteration.